src/lib/y2sap/products/do_install.rb

```diff
@@ -128,6 +128,13 @@ def run_script(script)
             /var/tmp and /var/adm/autoinstall/logs.")
         )
       end
+      if File.exist?("/var/run/sap-wizard/installationSuccesfullyFinished.dat")
+        File.open("/var/run/sap-wizard/installationSuccesfullyFinished.dat") do |file|
+          contents = file.read
+          Yast::Popup.ShowTextTimed("Installation Summary", contents, 100)
+          File.delete(file)
```

Comment:
I am not sure if deleting the file inside the `#open` block is correct. I would say it silently fails when trying to close the deleted file. I would delete it after the `#open` block:

~~~
file_name = "/var/run/sap-wizard/installationSuccesfullyFinished.dat"

if File.exists?(file_name)
  File.open(file_name) do |file|
    ...
  end
  File.delete(file_name)
end
~~~